README.md

```diff
@@ -1,67 +1,42 @@
+A demo chat application written in Vue.js for testing out the Ably Broadcaster for Laravel.
 
-<p align="center">
-<a href="https://travis-ci.org/laravel/framework"><img src="https://travis-ci.org/laravel/framework.svg" alt="Build Status"></a>
-<a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/dt/laravel/framework" alt="Total Downloads"></a>
-<a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/v/laravel/framework" alt="Latest Stable Version"></a>
-<a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/l/laravel/framework" alt="License"></a>
-</p>
+The application features the following:
+* Laravel built-in user authentication (login, registration)
+* Public chat rooms (no client permission to send messages, only broadcast supported)
+* Private chat rooms (chat between several participants)
+* Presence channels (visible join/leave events and the number of participants)
+* Sample implementation of a typing indicator
+* Joining multiple chat rooms simultaneously
 
-## About Laravel
+## Requirements
 
-Laravel is a web application framework with expressive, elegant syntax. We believe development must be an enjoyable and creative experience to be truly fulfilling. Laravel takes the pain out of development by easing common tasks used in many web projects, such as:
+This application uses Laravel 8, which requires PHP version 7.3 or greater.
 
-- [Simple, fast routing engine](https://laravel.com/docs/routing).
-- [Powerful dependency injection container](https://laravel.com/docs/container).
-- Multiple back-ends for [session](https://laravel.com/docs/session) and [cache](https://laravel.com/docs/cache) storage.
-- Expressive, intuitive [database ORM](https://laravel.com/docs/eloquent).
-- Database agnostic [schema migrations](https://laravel.com/docs/migrations).
-- [Robust background job processing](https://laravel.com/docs/queues).
-- [Real-time event broadcasting](https://laravel.com/docs/broadcasting).
+## Setup
 
-Laravel is accessible, powerful, and provides tools required for large, robust applications.
+1. Run `composer install` and `npm install` after cloning the project
+2. Create an `.env` file based on `.env.example`
+3. Set your `ABLY_KEY` and make sure the database credentials are correct
+4. Run `php artisan serve` in a terminal window to start the Laravel backend
+5. Run `npm run watch` in another terminal window
+6. Open http://127.0.0.1:8000 to access the demo app
 
-## Learning Laravel
+## Usage
 
-Laravel has the most extensive and thorough [documentation](https://laravel.com/docs) and video tutorial library of all modern web application frameworks, making it a breeze to get started with the framework.
```

Comment:
The web app runs in two modes 
1. Guest Mode ( Only public rooms can be created/accessed)
2. User sign-up ( Private and Presence rooms can be created/accessed)